src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {View} from 'react-native';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import useLocalize from '@hooks/useLocalize';
+import useSingleExecution from '@hooks/useSingleExecution';
+import Navigation from '@libs/Navigation/Navigation';
+import type {Route} from '@src/ROUTES';
+
+const advancedFilters = [
+    {
+        fieldName: 'type',
+        description: 'common.type' as const,
+        route: '/search/filters/type' as Route,
```

Comment:
Same for the the date route
```suggestion
        route: ROUTES.SEARCH_ADVANCED_FILTERS_TYPE,
```